gae_serve/main.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START app]
+import json
+import logging
+import os
+
+import pickle
+from google.cloud import storage
+from flask import Flask, request
+
+MODEL_BUCKET = os.environ['MODEL_BUCKET']
+MODEL_FILENAME = os.environ['MODEL_FILENAME']
+MODEL = None
+
+app = Flask(__name__)
+
+# NOTE: the first response could be slower if the model is large.
```

Comment:
This can be an issue - need to experiment what happens if the model takes more time to load.  GAE has a 30 seconds timeout for each request.

One possible approach is to use healthcheck to initiate model loading, and health check should not return OK until the model has been loaded.

Reply:
@dizcology : Oh, nice, I like the health check idea. Does the 30 second limit apply to health check?